src/Hintman/Suggestion/Core.hs

```diff
@@ -0,0 +1,24 @@
+module Hintman.Suggestion.Core
+       ( Line (..)
+       , Suggest (..)
+       , toLines
+       ) where
+
+
+data Line = Line
+    { lineRow  :: Int
+    , lineBody :: Text
+    } deriving (Eq, Show)
+
+data Suggest = Suggest
+    { suggestFile    :: FilePath
+    , suggestNew     :: Text
+    , suggestComment :: Maybe Text  -- ^ Optional comment to the suggestion
+    } deriving (Eq, Show)
+
+toLines :: Text -> [Line]
+toLines txt = map (\(n,x) -> Line{lineRow = n, lineBody = x}) numberedLines
```

Comment:
If you would name the vars `lineRow` and `lineBoby` you can use `RecordWildCards` to build the `Line`:
```suggestion
toLines txt = map (\(lineRow, lineBody) -> Line{..}) numberedLines
```